src/framework/audio/internal/worker/equaliser.h

```diff
@@ -47,7 +47,7 @@ class Equaliser : public IFxProcessor
     unsigned int m_sampleRate = 0;
     bool m_active = true;
 
-    float m_gain = 0, m_frequency = 1'000.f, m_q = 1.f;
+    float m_gain = 0, m_frequency = 1000.f, m_q = 1.f;
```

Comment:
I really wonder why/how that ever compiled?

Reply:
It seems to be some c++14 fanciness:
https://stackoverflow.com/questions/54447208/why-can-i-separate-digits-with-an-apostrophe-in-c-but-not-c

On a similar note, I was wondering why MU4 dropped -std= flag when compiling (MU3 has -std=gnu++11). It works by luck because gcc defaults to c++14 (version 6 up to 10 it seems), and as such you shouldn't get a warning?

Reply:
At least `lupdate` stumbles accros this, so let's better avoid to use this particular feature

Reply:
MU4 switched to C++14, Guess that's why?

Reply:
But yes, better make this explicit

Reply:
From https://en.cppreference.com/w/cpp/language/integer_literal:

> Optional single quotes (') may be inserted between the digits as a separator. They are ignored by the compiler. (since C++14)

Reply:
Yes, that's what @lyrra said already, basically. So I learen a new C++14 feature today ;-)
And as per my son (student of computer science) C23 will have this feature too.
I don't think this is really needed here though, and apparently throws off `lupdate`, so we'd better avoid using it as long as there are no compelling reasons.